src/tests/functional/plugin/shared/src/main.cpp

```diff
@@ -32,6 +34,10 @@ int main(int argc, char *argv[]) {
                 throw std::runtime_error("Incorrect value of \"--save_report_timeout\" argument");
             }
             ov::test::utils::OpSummary::setSaveReportTimeout(timeout);
+        } else if (std::string(argv[i]).find("--device_suffix") != std::string::npos) {
+            std::string deviceSuffix = std::string(argv[i]).substr(std::string("--device_suffix").length() + 1);
```

Comment:
Why cannot we just use -d GPU.0 instead of this additional argument?

Reply:
Using suffix looks comfortable, because we limit using func test binary for other devices

Reply:
Looks like it doesn't understand on the question why do we need this additional parameter

Reply:
to reuse this binary for the similar other device (GPU.0, GPU.1 and etc)

Reply:
discussed offline with @ilyachur and @vladimir-paramuzov